examples/defaulter-gen/output_tests/pointer/zz_generated.go

```diff
@@ -0,0 +1,45 @@
+// +build !ignore_autogenerated
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This file was autogenerated by defaulter-gen. Do not edit it manually!
+
+package pointer
+
+import (
+	runtime "k8s.io/apimachinery/pkg/runtime"
```

Comment:
this is not healthy unfortunately. It leads to a kube dependency in gengo.

Workaround: rename output_tests to _output_tests.

Reply:
If we rename output_tests to _output_tests, it will don't get the import `k8s.io/apimachinery`?

Reply:
We do the same at https://github.com/kubernetes/code-generator/tree/master/_examples. Vendoring tools (and go) ignore underscore folders.